### PR TITLE
AMD compatibility for dojox/Lightbox (fixes #18582)

### PR DESCRIPTION
**Pull request by AndrewGrossman**

### CODE REVIEW

File: image/Lightbox.js [hunk 1]
```diff
@@ -194,7 +194,7 @@ define(["dojo", "dijit", "dojox", "dojo/text!./resources/Lightbox.html", "dijit/
 
 		// errorImg: Url
 		//		Path to the image used when a 404 is encountered
-		errorImg: dojo.moduleUrl("dojox.image","resources/images/warning.png"),
+		errorImg: require.toUrl("dojox/image/resources/images/warning.png"),
```
Comment (neonstalwart) [inline image/Lightbox.js]: these things should probably be changed:

* add `require` as a dependency to this module
* use a relative path to reference the image

Comment (AndrewGrossman) [inline image/Lightbox.js]: Fixed.

**Comment by AndrewGrossman** [comment]
Suggestions from @neonstalwart have been applied.

**Comment by neonstalwart** [comment]
while it's not a complete conversion to AMD, it's better than it was.  thanks.

**Pull request merged**